engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -240,19 +240,19 @@ class CromwellApiServiceSpec extends AsyncFlatSpec with ScalatestRouteTest with
       }
   }
 
-  it should "return 500 when invalid workflow id is submitted to onHoldToSubmitted API end point" in {
+  it should "return 404 when invalid workflow id is submitted to onHoldToSubmitted API end point" in {
     val id = UnrecognizedWorkflowId
     Post(s"/workflows/$version/$id/releaseHold") ~>
       akkaHttpService.workflowRoutes ~>
       check {
         assertResult(
           s"""{
-             |  "status": "error",
+             |  "status": "fail",
```

Comment:
ToL: is there a big difference between `error` and `fail`? It seems slightly awkward for users to have to handle two synonymous but different `status` strings?

Reply:
So AFAIK, fail indicates that the data submitted in the request is incorrect, while error is when some kind of exception is thrown while processing the request.

Reply:
Ah this means FAIL in the all-caps sense.